src/Eviden.VirtualGrocer.sln

```diff
@@ -5,8 +5,12 @@ VisualStudioVersion = 17.5.33530.505
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "config", "config", "{64EE2856-049C-4CAB-B732-A8D0E77B96AF}"
 	ProjectSection(SolutionItems) = preProject
+		config\AppCreationScripts.md = config\AppCreationScripts.md
```

Comment:
I think we can drop these, because I'm moving the creation of these into the arm template

Reply:
@jgdestefano587 Still need the common.appsettings files, just wanted to drop the files we were adding.

Reply:
Putting those files back in now.